src/Tribe/Views/V2/Template/Excerpt.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Handles the manipulation of the excerpt.
+ *
+ * @since   4.9.10
+ *
+ * @package Tribe\Events\Views\V2\Template
+ */
+
+namespace Tribe\Events\Views\V2\Template;
+use Tribe__Template as Base_Template;
+use Tribe__Events__Main as Plugin;
+
+/**
+ * Class Excerpt
+ *
+ * @since   4.9.10
+ *
+ * @package Tribe\Events\Views\V2\Template
+ */
+class Excerpt extends Base_Template {
+
+	/**
+	 * Excerpt constructor.
+	 *
+	 * @since 4.9.10
+	 */
+	public function __construct() {
+		$this->set_template_origin( Plugin::instance() );
+		$this->set_template_folder( 'src/views/v2' );
+		$this->set_template_context_extract( true );
+		$this->set_template_folder_lookup( true );
+	}
+
+	/**
+	 * Filters the excerpt length.
+	 *
+	 * Set the excerpt length for list and day view.
+	 *
+	 * @since 4.9.10
+	 *
+	 * @param int $length The excerpt length.
+	 *
+	 * @return int The excerpt length modified, if necessary.
+	 */
+	public function maybe_filter_excerpt_length( $length ) {
+		$context = tribe_context();
+		$view = $context->get( 'event_display_mode', 'list' );
+
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 4 spaces but found 1 space

<sub>hash: eb552c6855c1bfde2eebf26682481c26</sub>